pyvista/plotting/plotting.py

```diff
@@ -2599,30 +2609,43 @@ def get_image_depth(self,
 
         Parameters
         ----------
-        fill_value : float
-            Fill value for points in image that don't include objects in scene.
-            To not use a fill value, pass ``None``.
+        fill_value : float, optional
+            Fill value for points in image that do not include objects
+            in scene.  To not use a fill value, pass ``None``.
 
-        reset_camera_clipping_range : bool
-            Reset the camera clipping range to include data in view?
+        reset_camera_clipping_range : bool, optional
+            Reset the camera clipping range to include data in view.
 
         Returns
         -------
         image_depth : numpy.ndarray
-            Image of depth values from camera orthogonal to image plane
+            Image of depth values from camera orthogonal to image
+            plane.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> plotter = pyvista.Plotter()
+        >>> _ = plotter.add_mesh(pyvista.Sphere())
+        >>> plotter.show()  # doctest:+SKIP
+        >>> zval = plotter.get_image_depth()  # doctest:+SKIP
```

Comment:
As is, this raises due to the missing `store_image`:
```suggestion
        >>> plotter.store_image = True
        >>> plotter.show()  # doctest:+SKIP
        >>> zval = plotter.get_image_depth()  # doctest:+SKIP
```

And on a related note, can we remove the doctest skips to make this tested?

Reply:
> And on a related note, can we remove the doctest skips to make this tested?

Actually, yes.  I know we had that disabled at some point for some reason, but I've enabled that again in this PR and it works out.  Lots of change to make...

Reply:
Oof, I just meant this one instance :sweat_smile: